### DISCUSSION TITLE
Egg Timer Not Behaving as Expected

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#948] [Q&A] [answered]
I have a one-body pool with a fairly straightforward Nixie & REM Chem setup--Intelliiflow VS pump, Polaris cleaner, Intellichlor salt cell,  acid pump, gas heater, pool lights, and one actuated valve.  The actuated valve is for an drain line that's used only when we have a rain deluge (like today!).

I use a Zigbee moisture sensor glued to the underside of the pool filler lid to determine when the passive drain line cannot keep up with the rain.  When the moisture sensor is triggered, Home Assistant, starts the filter pump if it's not running,  opens the actuated valve for 10 minutes, closes the valve, & restores the pump to its previous state (off, it it's  the middle of the night, for example).   Thanks to @Crewski for a superb HA integration!

To minimize the likelihood of the drain line not closing after 10 minutes (due to HA going down, for example), I have an egg timer for the actuated valve.  However, I have done something incorrectly and the egg timer does not turn off the overflow valve when I actuate it "manually" via the dashPanel, as I did in the wee hours this morning.  Do I need to check the "Don't Stop" option, for example?  I don't know how that option affects the egg timer.

Below are screenshots of the valve configuration.  

Can you please tell me what I'm doing incorrectly vis a vis, the actuated valve's egg timer?  Thank you for your help!!!!

![image](https://github.com/tagyoureit/nodejs-poolController/assets/41341956/dd59a917-c678-44ab-8d45-1326ce58f17c)

![image](https://github.com/tagyoureit/nodejs-poolController/assets/41341956/3488b08b-f2c6-4062-b940-fce46b78dc49)

**Comment by tagyoureit** [comment, marked as answer]
It would be helpful if you could provide a replay, if you get a chance.  Otherwise, I'll try to dig through the code and see if I can find anything.

One thing you should update first is that you have both the Circuit and the Valve connected to the same REM "Drain Line".  You shouldn't have multiple pool features controlling the same physical equipment.
You want `Circuit->Valve->REM`.  
But not:
```
Circuit->REM
        \ 
         Valve->REM
```
AKA, if that wasn't clear get rid of the REM connection on the Aux-Circuit and let the Valve device control the REM physical valve.

See if this helps with the egg timer issue.

**Comment by FlaMike** [comment]
Thank you for your reply!  I will update the config as you suggest & see if
that takes care of things & get back to you over the weekend.

**Reply by FlaMike** [reply]
I removed REM as the controller for the Aux-Circuit as you suggested & set the egg timer to 1 minute so I didn't drain a pool that wasn't overflowing :-)  I then turned on the drain line from dashPanel & watched things for a minute...  It worked!!!!!  The egg timer performed as expected & closed the drain line.  I then restored the egg timer to 10 minutes & saved the circuit.

Case closed!   Thank you for your help, as always!!!!!   Have a super weekend--mine is getting better by the, uh, minute!
